client/src/templates/Guide/components/GuideFooter.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link, Spacer } from '../../../components/helpers';
+
+const propTypes = {
+  githubPath: PropTypes.string
+};
+class GuideFooter extends React.Component {
```

Comment:
let's redo this to a function component

Reply:
I believe the new changes should resolve this.